[2.7] Corrige seeds relativo a obtenção do `dbname`

msoares94 opened the pull request:
**DESCRIÇÃO:**

Foi corrigindo as seeds para contemplar o valor padrão para os atributos:
```
legacy.app.database.hostname=localhost
legacy.app.database.port=5432
legacy.app.database.dbname=ieducar
legacy.app.database.username=ieducar

```

Além de habilitar a edição pela interface da aplicação, permitindo assim configurar o tenent para ser utilizado pela aplicação no momento de registar no storage

Este PR corrigi o erro que cometi na correção da [ISSUE-777:](https://github.com/portabilis/i-educar/issues/777), na correção dessa ISSUE, eu substitui os valores acima por `********`, com o intuito de mascara o valor na interface da aplicação e isso tem impacto direto de como a aplicação trabalha com o tenant no momento de upload/downloas de arquivos em instalação com tenant.

**AMBIENTE:**

- Plataforma utilizada (Docker e Instalação direta)
- Sistema operacional e versão (Ubuntu 20.04)

msoares94 committed:
Corrigindo migrations relativo a parte de tenant para novas instalações do i-Educar.

edersoares commented on database/migrations/2020_08_03_151235_populate_settings_categories_table.php:
```diff
@@ -33,7 +33,7 @@ public function up()
             [
                 'name' => 'Banco de dados',
-                'enabled' => false,
+                'enabled' => true,
```
Por padrão esta seção deve vir desabilitada para evitar alterar as informações do banco de dados. Isso deve ser habilitado na sua instalação.

msoares94 replied:
Entendido @edersoares, obrigado por informar!

msoares94 committed:
Update 2020_08_03_151235_populate_settings_categories_table.php

Trocando o valor de true para false, pois por padrão esta seção deve vir desabilitada para evitar alterar as informações do banco de dados. Isso deve ser habilitado na instalação.
